src/main/resources/locales/en-US.yml

```diff
@@ -1231,6 +1231,10 @@ protection:
       description: '&a Toggle hive harvesting.'
       name: Hive harvesting
       hint: Harvesting disabled
+    HURT_TAMED_ANIMALS:
+      description: Toggle hurting
+      name: Hurt tamed animals
```

Comment:
If it is enabled, can you hurt tabled animals, or are you not allowed? We've had a few flags like this one where it is ambiguous so I recommend changing this text to be more explicit in what enabled/disables means exactly.